diff --no-dereference -urb wanpipe-7.0.34/patches/kdrivers/src/wanrouter/af_wanpipe.c wanpipe-7.0.34b/patches/kdrivers/src/wanrouter/af_wanpipe.c
--- telephony-kmods-a/wanpipe-7.0.34/patches/kdrivers/src/wanrouter/af_wanpipe.c	2008-08-01 13:00:00.000000000 +0000
+++ telephony-kmods-b/wanpipe-7.0.34/patches/kdrivers/src/wanrouter/af_wanpipe.c	2022-05-16 12:54:43.675801232 +0000
@@ -600,7 +600,7 @@
 		return -EPROTOTYPE;
 
 	add_wait_queue(WAN_SK_SLEEP(sk),&wait);
-	current->state = TASK_INTERRUPTIBLE;
+	WRITE_ONCE(current->__state, TASK_INTERRUPTIBLE);
 	for (;;){
 		skb = skb_dequeue(&sk->sk_receive_queue);
 		if (skb){
@@ -619,7 +619,7 @@
 		}
 		schedule();
 	}
-	current->state = TASK_RUNNING;
+	WRITE_ONCE(current->__state, TASK_RUNNING);
 	remove_wait_queue(WAN_SK_SLEEP(sk),&wait);
 
 	if (err != 0)
